tests/test_servo.py

```diff
@@ -231,25 +232,23 @@ def test_load_configuration(connect_to_slave, read_config, pytestconfig):
     for register in loaded_registers:
         reg_id = register.attrib.get("id")
         storage = register.attrib.get("storage")
-        access = register.attrib.get("access")
-        if storage is None or access != "rw":
-            continue
         subnode = int(register.attrib.get("subnode"))
         dtype = register.attrib.get("dtype")
 
-        if reg_id in servo.dictionary.registers(subnode):
-            if (
-                servo.dictionary.registers(subnode)[reg_id].address_type
-                == REG_ADDRESS_TYPE.NVM_NONE
-            ):
-                continue
+        # Check if the register exists in the drive
+        try:
             value = servo.read(reg_id, subnode=subnode)
-            if dtype == "str":
-                assert value == storage
-            elif dtype == "float":
-                assert value == pytest.approx(float(storage), 0.0001)
-            else:
-                assert value == int(storage)
+        except ILIOError:
```

Comment:
How could a config that you just saved contain a register that doesn't exist?

Reply:
The eoe dictionary contains Ethernet-related registers that don't exist in the EVE-XCR-E drive

Reply:
they don't exist via ethercat?
Or they don't exist at all? If they don't exist at all create issue on VOLCANO-490

Reply:
The registers don't exist on the EVE-XCR-E (they do on the CAP-XCR-E). When attempting to read them, an SDO "object doesn't exist" abort error is thrown.

Reply:
I mean if they exist on EoE. Either way I'm creating it to review: VOLCANO-615